wineCollection_2020-09-28.sql

```diff
@@ -0,0 +1,139 @@
+# ************************************************************
+# Sequel Pro SQL dump
+# Version 4541
+#
+# http://www.sequelpro.com/
+# https://github.com/sequelpro/sequelpro
+#
+# Host: 127.0.0.1 (MySQL 5.7.31)
+# Database: wineCollection
+# Generation Time: 2020-09-28 10:59:48 +0000
+# ************************************************************
+
+
+/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
+/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
+/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
+/*!40101 SET NAMES utf8 */;
+/*!40014 SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0 */;
+/*!40101 SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='NO_AUTO_VALUE_ON_ZERO' */;
+/*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;
+
+
+# Dump of table brands
+# ------------------------------------------------------------
+
+DROP TABLE IF EXISTS `brands`;
+
+CREATE TABLE `brands` (
+  `id` int(11) unsigned NOT NULL AUTO_INCREMENT,
+  `NameOfBrand` varchar(100) DEFAULT NULL,
+  PRIMARY KEY (`id`)
+) ENGINE=InnoDB DEFAULT CHARSET=latin1;
+
+LOCK TABLES `brands` WRITE;
+/*!40000 ALTER TABLE `brands` DISABLE KEYS */;
+
+INSERT INTO `brands` (`id`, `NameOfBrand`)
+VALUES
+	(1,'Bread & Butter'),
+	(2,'Wolf Blass'),
+	(3,'McGuigan'),
+	(4,'Campo Viejo'),
+	(5,'Sainsbury'),
+	(6,'Waitrose');
+
+/*!40000 ALTER TABLE `brands` ENABLE KEYS */;
+UNLOCK TABLES;
+
+
+# Dump of table Cost
+# ------------------------------------------------------------
+
+DROP TABLE IF EXISTS `Cost`;
+
+CREATE TABLE `Cost` (
+  `id` int(11) unsigned NOT NULL AUTO_INCREMENT,
+  `cost` float DEFAULT NULL,
+  PRIMARY KEY (`id`)
+) ENGINE=InnoDB DEFAULT CHARSET=latin1;
+
+LOCK TABLES `Cost` WRITE;
+/*!40000 ALTER TABLE `Cost` DISABLE KEYS */;
+
+INSERT INTO `Cost` (`id`, `cost`)
+VALUES
+	(1,7.99),
+	(2,15.99),
+	(3,8),
+	(4,7.5);
+
+/*!40000 ALTER TABLE `Cost` ENABLE KEYS */;
+UNLOCK TABLES;
+
+
+# Dump of table country
+# ------------------------------------------------------------
+
+DROP TABLE IF EXISTS `country`;
+
+CREATE TABLE `country` (
+  `id` int(11) unsigned NOT NULL AUTO_INCREMENT,
+  `country` varchar(400) DEFAULT NULL,
```

Comment:
varchar is only optimised upto 255 - unlikely you will have a country longer than than, so set that to the length